auth/auth.go

```diff
@@ -787,12 +787,12 @@ func (auth *Authenticator) AuthenticateUser(username string, password string) (U
 func (auth *Authenticator) AuthenticateUntrustedJWT(rawToken string, oidcProviders OIDCProviderMap, localJWT LocalJWTProviderMap, callbackURLFunc OIDCCallbackURLFunc) (User, PrincipalConfig, error) {
 	token, err := jwt.ParseSigned(rawToken)
 	if err != nil {
-		base.DebugfCtx(auth.LogCtx, base.KeyAuth, "Error parsing JWT in AuthenticateUntrustedJWT: %v", err)
+		base.InfofCtx(auth.LogCtx, base.KeyAuth, "Error parsing JWT in AuthenticateUntrustedJWT: %v", err)
```

Comment:
All of this AuthenticateUntrustedJWT logging It looks like this could be logged based on malformed JWTs provided by clients.  Given that, I don't know if we want spam info logs with these messages if a bad/malicious client is spamming authentication requests.

Reply:
I removed these because I don't really know, but I thought about putting them in so you'd know if there was a bad/malicious client and what they were doing wrong so you could block them.